.github/workflows/ci.yml

```diff
@@ -20,6 +20,7 @@ jobs:
           -G "Unix Makefiles" \
           -DCMAKE_BUILD_TYPE=Debug \
           -DBUILD_CLONE_SUBMODULES=ON \
+          -DUNITTEST=1 \
```

Comment:
This question may be out of the scope of this PR.
Do we need to add `UNITTEST` variable in other library as well?

Reply:
Which other library are you talking about?